docs/resourceTuning/project-resource-quotas.md

```diff
@@ -14,24 +14,60 @@ tags:
 description: The quota based resource allocation for project sets in Openshift 4 Platform.
 ---
 
-All project sets in the new OpenShift Container Platform (OCP) 4 Platform are provisioned by default with a "small" resource quota size that includes a certain amout of CPU, RAM and Storage.  The amount of resources within each quota size - small, medium and large are as follows:
+All project sets in the new OpenShift Container Platform (OCP) 4 Platform are provisioned by default with a set of resource quotas that include a certain amout of CPU, RAM and Storage:
 
 **CPU/RAM/Storage resource quotas (per namespace)**:
-* **Small (Default)**:<br/> 
   CPU: 4 cores as request, 8 cores as limit <br/>
   RAM: 16GBs as request, 32GBs as limit<br/>
   20 PVC count , 50Gbs overall storage with 25 GBs for backup storage<br/>
```

Comment:
Our default size will have only 0.5-1.5 CPU, 2-4GiB Memory, and 1Gi Storage, 521m backup, 60 PVC and 5 snapshot